pkg/server/spec/cloudsqlproxy.go

```diff
@@ -55,3 +62,26 @@ func newCloudSQLProxyContainerArgs(csp *CloudSQLProxy) []string {
 func newCloudSQLProxyContainerMountPath(csp *CloudSQLProxy) string {
 	return path.Join("/secrets/cloudsql", path.Base(csp.CredentialFile))
 }
+
+func newCloudSQLProxyContainerInstanceString(csp *CloudSQLProxy, a *app.App) string {
+	instanceName := "%s:%s:%s=tcp:3306"
+	if csp.Instances == "" {
+		dbProject := ""
```

Comment:
What do you think about adding unit tests for this function? There is neither direct tests for the function nor tests for the specific uses cases (DB_PROJECT, DB_ZONE and DB_NAME).

Reply:
[https://codecov.io/gh/luizalabs/teresa/pull/588/diff#D1-65](full coverage :tada:)